src/index.js

```diff
@@ -64,7 +64,7 @@ class QueryBuilder extends BaseBuilder {
 	* @return An elasticsearch query
```

Comment:
Document use case on why this needs to be done (when mixing with musts/must_nots, etc)